src/app/controllers/quotes-controller.ts

```diff
@@ -0,0 +1,93 @@
+import _quotes from '../../data/quotes.json';
+import { IQuote } from '../models/quotes.model';
+import { v4 as uuidv4 } from 'uuid';
+
+const generateQuotes = () => {
+    let num = 0;
+    return _quotes.map(quote => {
+        num++;
+        return {
+            ...quote,
+            id: num.toString(),
+            isDeleted: false,
+            tags: quote.tags?.split(', '),
+        };
+    });
+}
+const QUOTES: IQuote[] = generateQuotes();
+
+
+const getQuotesList = (req, res) => {
```

Comment:
Better to separate the framework from your controller. Because if you want to switch to a different library, it will be a problem to change the code in different places